src/System.IO.Packaging/src/System.IO.Packaging.csproj

```diff
@@ -9,17 +9,18 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <RootNamespace>System.IO.Packaging</RootNamespace>
     <AssemblyName>System.IO.Packaging</AssemblyName>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
```

Comment:
Why did you remove this? I would expect it should stay but you should add the condition `'$(PackageTargetFramework)' == ''`